### ISSUE TITLE
Case sensitive sort

### ISSUE THREAD

**Issue by dandanknight** [balderdashy/waterline#1207]
First, congrats on the amazing work.

I'm implementing some server side pagination, sorting, and filtering, using sails blueprints. I've noticed that when sorting, I get Uppercase first, then lowercase afterwards. I believe that this is an issue when getting sorts from Mongo (which I will ultimately be doing), but, it's also happening using 'sails-disk'.

Is this behaviour expected, and is there anything that can be done other than resorting the collection after it's returned from the ORM/DB?

**Comment by devinivy** [comment]
It's hard, because it depends on the underlying database in certain cases.  For example, the collation of a mysql column affects this.  In the case of mongo, I'm not entirely sure!  There are not collation options like this that are intrinsic to waterline.  I think some users have experimented with a case-insensitive feature that is dormant and not "officially" supported.  I don't know much about it– maybe someone can chime in!

**Comment by sailsbot** [comment]
Thanks for posting, @dandanknight.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
